src/ashley/templates/forum_conversation/topic_detail.html

```diff
@@ -0,0 +1,121 @@
+{% extends 'board_base.html' %}
```

Comment:
I agree with you I had trouble at first to see this code that seems to be repeated. That would be a nice next step :-) I hope I can still validate this PR first :-)

Reply:
Well this PR is 99% ready to merge imo but if you think you can get these blocks in a release in a few days maybe you should wait for it? Otherwise, we can just open an issue and list all the templates that we want to stop overriding in full (I think I remember seeing other cases in Ashley already). Then the PR to django-machina could cover all templates :taco:

Reply:
Either way I would have to rewrite some code and I have no influence regarding when this work could be accepted in django machina. I would propose :-) for both project to advance to validate this PR in our project, submit the work in machina and when this is done to do a new PR on our project. What do you think of this strategy ?

Reply:
Yes :+1: let's open an issue in Ashley listing all templates that we have overriden and would want to modify in django-machina.